specification/columns/consumedunit.md

```diff
@@ -5,7 +5,7 @@ The Consumed Unit represents a provider-specified measurement unit indicating ho
 The ConsumedUnit column adheres to the following requirements:
 
 * ConsumedUnit MUST be present in the billing data when the provider supports the measurement of usage.
-* ConsumedUnit MUST be of type String, and the units of measure used in ConsumedUnit SHOULD adhere to the values and format requirements specified in the [UnitFormat](#unitformat) attribute.
+* ConsumedUnit MUST be of type String, and the units of measure used in ConsumedUnit SHOULD adhere to the values and format requirements specified in the [Unit Format](#unitformat) attribute.
```

Comment:
should be normative text therefore attribute id not attribute name?

Reply:
**Heads up:** 
For the sake of consistency, and based on the discussion in the maintainers' meeting on Oct 8th, we'll be switching **from attribute names to attribute IDs in the normative paragraphs**, while retaining attribute names in the Content Constraints tables.

Reply:
suggestion addressed. resolving conversation.